default.nix

```diff
@@ -15,28 +15,30 @@ let
     '';
   };
 
+  helmWithPlugins = pkgs.wrapHelm pkgs.kubernetes-helm {
+    plugins = with pkgs.kubernetes-helmPlugins; [ helm-s3 helm-secrets helm-diff ];
```

Comment:
This feels a bit redundant with the helm plugin stuff in cailleach. Can we agree where to put this? ;-)